src/java.base/share/classes/java/io/Console.java

```diff
@@ -161,27 +167,65 @@ public Reader reader() {
      *         limited by the maximum dimension of a Java array as defined by
      *         <cite>The Java Virtual Machine Specification</cite>.
      *         The behaviour on a
-     *         {@code null} argument depends on the <a
-     *         href="../util/Formatter.html#syntax">conversion</a>.
+     *         {@code null} argument depends on the {@link
+     *         Formatter##syntax conversion}.
      *
      * @throws  IllegalFormatException
      *          If a format string contains an illegal syntax, a format
      *          specifier that is incompatible with the given arguments,
      *          insufficient arguments given the format string, or other
      *          illegal conditions.  For specification of all possible
-     *          formatting errors, see the <a
-     *          href="../util/Formatter.html#detail">Details</a> section
+     *          formatting errors, see the {@link
+     *          Formatter##detail Details} section
      *          of the formatter class specification.
      *
      * @return  This console
      */
-    public Console format(String fmt, Object ...args) {
+    public Console format(String format, Object ... args) {
+        throw newUnsupportedOperationException();
+    }
+
+    /**
+     * Writes a formatted string to this console's output stream using
+     * the specified format string and arguments with the specified
+     * {@code locale}.
+     *
+     * @param  locale locale used for formatting
+     * @param  format
+     *         A format string as described in {@link
+     *         Formatter##syntax Format string syntax}.
+     *
+     * @param  args
+     *         Arguments referenced by the format specifiers in the format
+     *         string.  If there are more arguments than format specifiers, the
+     *         extra arguments are ignored.  The number of arguments is
+     *         variable and may be zero.  The maximum number of arguments is
```

Comment:
readLine and readPassword don't have this statement (The number of arguments is variable and may be zero). Is this statement helpful for those methods as well?

Reply:
Modified each description for `args` consistently.